### PR TITLE
Misc 24.05 updates

### PR DESCRIPTION
**Pull request by brooksdavis**

### CODE REVIEW

File: src/morello-install/README.md [hunk 1]
```diff
@@ -670,28 +677,27 @@ Otherwise, press Enter to complete the installation:
...
-                     ├───────────────────────────────────┤
-                     │ [ Reboot ] [Shutdown] [Live CD ]  │
-                     └───────────────────────────────────┘
+                  ┌───────────────┤Complete├────────────────┐
+                  │ Installation of FreeBSD complete! Would │
+                  │ you like to reboot into the installed   │
+                  │ system now?                             │
+                  ├─────────────────────────────────────────┤
+                  │[  Reboot   ] [ Shutdown  ] [Live System]│
+                  └─────────────────────────────────────────┘
```
Comment (brooksdavis) [inline src/morello-install/README.md:700]: The "Logging in" section below needs an updated, but that depends on the final release image.

**Review by dpgao — APPROVED** [review]

File: src/features/c18n.md [hunk 2]
```diff
@@ -51,45 +51,68 @@ off-the-shelf aarch64c binaries and libraries can be used with library
...
+sysctl security.cheri.lib_based_c18n_default=1
 ```
 
-For example, compile `helloworld.c` using:
+## Enabling library compartmentalization temporarily for a particular binary
 
+To run a particular binary with library compartmentalization enabled just for
+this run, use `proccontrol`:
+
+`proccontrol -m cheric18n -s enable ./helloworld`
```
Comment (jrtc27) [inline src/features/c18n.md:74]:
````suggestion
```
proccontrol -m cheric18n -s enable ./helloworld
```
````

File: src/features/kernel.md [hunk 3]
```diff
@@ -1,10 +1,17 @@
 # Kernel compilation modes
 
-The CheriBSD kernel can be compiled either as hybrid or pure-capability code:
+The CheriBSD kernel can be compiled either as hybrid, pure-capability,
+or benchmark code:
```
Comment (jrtc27) [inline src/features/kernel.md:4]:
```suggestion
The CheriBSD kernel can be compiled as hybrid, pure-capability,
or benchmark code:
```

File: src/morello-install/README.md [hunk 4]
```diff
@@ -640,28 +651,24 @@ If there are further configuration settings to change, select an appropriate
 option from the menu and press Space.
```
Comment (jrtc27) [inline src/morello-install/README.md:651]: This needs updating now the Exit entry has moved to its own separate Finish button

File: src/nonfeatures/README.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ Many other kernel features are not yet well validated, including:
 
 ## Alpha ZFS support
 
-The 23.11 release includes Alpha support for the ZFS file system.  It is
+The 24.05 release includes Alpha support for the ZFS file system.  It is
 lightly tested, but works without known issues with a hybrid kernel.  With
 the pure capability kernel, filesystems can be created and used, but it must
 not be the root file system as it currently hangs on boot.
```
Comment (jrtc27) [inline src/nonfeatures/README.md:17]:
```suggestion
lightly tested, but works without known issues.
```

File: src/morello-install/README.md [hunk 6]
```diff
@@ -640,28 +651,24 @@ If there are further configuration settings to change, select an appropriate
...
 Otherwise, press Enter to complete the installation:
```
Comment (gvnn3) [inline src/morello-install/README.md:652]:
```suggestion
Otherwise, press navigate to the Finish button and press Enter to complete the installation:
```

Comment (jrtc27) [reply src/morello-install/README.md:652]: I believe Finish is selected by default, and if you want to choose one of the list items you have to also navigate left to Select. It’s a bit confusing and I’m not sure it’s an improvement over the previous interface.

Comment (gvnn3) [reply src/morello-install/README.md:652]: OK, but we're not about to change the button now, so, how do we best explain it?

Comment (jrtc27) [reply src/morello-install/README.md:652]: See the first part of the message?..

Comment (gvnn3) [reply src/morello-install/README.md:652]: Better?

Comment (brooksdavis) [reply src/morello-install/README.md:652]: Wow, the new UI is horrible

**Comment by brooksdavis** [comment]
I've adjusted the wording of the Final Configuration section in a new commit for ease of review and curated the GitHub UI generated commits

**Pull request merged**